model/database.php

```diff
@@ -175,4 +175,25 @@ function testDatabase()
 
     }
 
+    /** This fetches all the information to be displayed for the tutors view.
+     * Only important data is chosen.
+     * @param int $year the year that is required
+     * @param int $userID the database id of the tutor
+     * @return array returns all the required checkboxes as well as the name
+     * @author oleg
+     */
+    function getTutorsChecklist($year, $userID){
```

Comment:
I think this function is redundant, Dallas wrote one called getTutor that does the same thing.

Reply:
his gets extra information that I do not use. Is that fine?

Reply:
yeah, may be redundant, but we can keep until we decide as a group if we wan't to keep or not.